docs/migration/00c_Vibrational_structure_with_v0.5.ipynb

```diff
@@ -0,0 +1,1180 @@
+      "+ (52.162006547054936+0j) * ( +_3_3 -_3_1 +_2_1 -_2_1 )\n",
+      "+ (24.58940569981309+0j) * ( +_3_3 -_3_1 +_2_2 -_2_0 )\n",
+      "+ (86.93667757842489+0j) * ( +_3_3 -_3_1 +_2_2 -_2_2 )\n",
+      "+ (27.60158927849401+0j) * ( +_3_3 -_3_2 +_2_0 -_2_1 )\n",
+      "+ (27.60158927849401+0j) * ( +_3_3 -_3_2 +_2_1 -_2_0 )\n",
+      "+ (19.517270950349022+0j) * ( +_3_3 -_3_2 +_2_1 -_2_2 )\n",
+      "+ (19.517270950349022+0j) * ( +_3_3 -_3_2 +_2_2 -_2_1 )\n",
+      "+ (-71.44123875000004+0j) * ( +_3_3 -_3_3 +_0_0 -_0_0 )\n",
+      "+ (-214.32371625000013+0j) * ( +_3_3 -_3_3 +_0_1 -_0_1 )\n",
+      "+ (-29.360095625000014+0j) * ( +_3_3 -_3_3 +_1_0 -_1_0 )\n",
+      "+ (298.7269117198203+0j) * ( +_3_3 -_3_3 +_1_0 -_1_1 )\n",
+      "+ (298.7269117198203+0j) * ( +_3_3 -_3_3 +_1_1 -_1_0 )\n",
+      "+ (-88.08028687500004+0j) * ( +_3_3 -_3_3 +_1_1 -_1_1 )\n",
+      "+ (49.688450000000024+0j) * ( +_3_3 -_3_3 +_2_0 -_2_0 )\n",
+      "+ (70.27007988329744+0j) * ( +_3_3 -_3_3 +_2_0 -_2_2 )\n",
+      "+ (149.06535000000008+0j) * ( +_3_3 -_3_3 +_2_1 -_2_1 )\n",
+      "+ (70.27007988329744+0j) * ( +_3_3 -_3_3 +_2_2 -_2_0 )\n",
+      "+ (248.4422500000001+0j) * ( +_3_3 -_3_3 +_2_2 -_2_2 )\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_nature.second_q.drivers import GaussianForcesDriver\n",
+    "from qiskit_nature.second_q.problems import HarmonicBasis\n",
+    "\n",
+    "driver = GaussianForcesDriver(logfile=\"aux_files/CO2_freq_B3LYP_631g.log\")\n",
+    "basis = HarmonicBasis(num_modals=[2, 2, 3, 4])\n",
+    "\n",
+    "# this is now done explicitly and already requires the basis\n",
+    "problem = driver.run(basis=basis)\n",
+    "problem.hamiltonian.truncation_order = 2\n",
+    "\n",
+    "hamiltonian = problem.hamiltonian\n",
+    "print(hamiltonian.second_q_op())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "21756d06",
+   "metadata": {},
+   "source": [
+    "## `qiskit_nature.drivers`\n",
+    "\n",
+    "This section deals exclusively with the migration of the vibrational structure-related drivers."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fc9d966f",
+   "metadata": {},
+   "source": [
+    "The table below summarizes where each of the vibrational structure components of `qiskit_nature.drivers.second_quantization` has ended up.\n",
+    "\n",
+    "| Legacy component | New location |\n",
+    "|:-----------------|:-------------|\n",
+    "| BaseDriver | `qiskit_nature.second_q.drivers.BaseDriver` |\n",
+    "| VibrationalStructureDriver | `qiskit_nature.second_q.drivers.VibrationalStructureDriver` |\n",
+    "| VibrationalStructureDriverType | **removed** |\n",
+    "| VibrationalStructureMoleculeDriver | **removed** |\n",
+    "| GaussianForcesDriver | `qiskit_nature.second_q.drivers.GaussianForcesDriver` |\n",
+    "| GaussianLogDriver | `qiskit_nature.second_q.drivers.GaussiaLogDriver` |\n",
+    "| GaussianLogResult | `qiskit_nature.second_q.drivers.GaussianLogResult` |\n",
```

Comment:
Strictly speaking these are not vibrational - the ForcesDriver would run things in a way to have the information come out of a log driver instance where it was vibrational. But I think they are ok here, at least they get a mention somewhere.